protocol/Makefile

```diff
@@ -51,11 +51,11 @@ js/rpc.js: build-stamp | config
 	cp ./js/engine-gen.json ../shared/actions/json
 
 build-go-bot-types: avdl/keybase1/*.avdl avdl/gregor1/*.avdl avdl/chat1/*.avdl avdl/stellar1/*.avdl $(AVDLC) | config
-	$(AVDLC) -b -l go -t -o ../../go-keybase-chat-bot/kbchat/types/keybase avdl/keybase1/*.avdl
-	$(AVDLC) -b -l go -t -o ../../go-keybase-chat-bot/kbchat/types/gregor avdl/gregor1/*.avdl
-	$(AVDLC) -b -l go -t -o ../../go-keybase-chat-bot/kbchat/types/chat avdl/chat1/*.avdl
-	$(AVDLC) -b -l go -t -o ../../go-keybase-chat-bot/kbchat/types/wallet avdl/stellar1/*.avdl
-	(cd ../../go-keybase-chat-bot/kbchat/types && go fmt ./...)
+	$(AVDLC) -b -l go -t -o ../../go-keybase-chat-bot/kbchat/types/keybase1 avdl/keybase1/*.avdl
+	$(AVDLC) -b -l go -t -o ../../go-keybase-chat-bot/kbchat/types/gregor1 avdl/gregor1/*.avdl
+	$(AVDLC) -b -l go -t -o ../../go-keybase-chat-bot/kbchat/types/chat1 avdl/chat1/*.avdl
+	$(AVDLC) -b -l go -t -o ../../go-keybase-chat-bot/kbchat/types/stellar1 avdl/stellar1/*.avdl
+	(cd ../../go-keybase-chat-bot/kbchat/types && goimports -w .)
```

Comment:
This Makefile is what I used to compile the types in go-keybase-chat-bot#14. I had to make a couple of name tweaks to match the package names. The compiler also spits out imports relative to `github.com/keybase/client`, so I opted to use `goimports` to fix that issue until I update the compiler to support relative imports.